### ISSUE TITLE
[Question] Is it possible to calculate IV if the option price is given?

### ISSUE THREAD

**Issue by ghost** [vollib/py_vollib#24]
Is it possible to calculate implied volatility if the option price is given?

**Comment by mainTrim13** [comment]
I wrote a routine to do this. 
Basically, you run the model and change the IV until you reach your desired price. 1) You need an IV value to begin from. 2) There are limits to the equation. So as your IV values get very large or very small the price of the option becomes asymptotic and creates a pseudo infinite loop.  But I find it to be a valuable tool because the only thing you can lean on is the bid/ask price. Everything else is theoretical and therefore relative to your model.

**Comment by lbrichards** [comment]
Yes, this is supported. The implied-volatility functions take the observed option price as the first argument.

For example, with Black-Scholes:

```python
from vollib.black_scholes import black_scholes
from vollib.black_scholes.implied_volatility import implied_volatility

flag = "c"
S = 100
K = 100
t = 0.5
r = 0.01
sigma = 0.2

price = black_scholes(flag, S, K, t, r, sigma)
iv = implied_volatility(price, S, K, t, r, flag)

print(price)  # 5.876024233827604
print(iv)     # 0.2
```

Equivalent functions are also available for Black and Black-Scholes-Merton:

```python
from vollib.black.implied_volatility import implied_volatility
from vollib.black_scholes_merton.implied_volatility import implied_volatility
```

I have added a README example for this in the upcoming official release.
